src/main/kotlin/solve/scene/model/LayerSettings.kt

```diff
@@ -10,17 +13,30 @@ sealed class LayerSettings(val name: String) {
     val colorManager = ColorManager<Long>()
 
     class PointLayerSettings(name: String, private val layerColorManager: ColorManager<String>) : LayerSettings(name) {
-        var color = layerColorManager.getColor(name)
+        companion object {
+            private const val OrdinaryRadius: Double = 5.0
+        }
+
+        val selectedRadiusProperty = SimpleObjectProperty(OrdinaryRadius)
+        var selectedRadius: Double
+            get() = selectedRadiusProperty.get()
+            set(value) = selectedRadiusProperty.set(value)
+
+        val colorProperty = SimpleObjectProperty(layerColorManager.getColor(name))
+        var color: Color
```

Comment:
> Redundant property, use only SimpleObjectProperty and its changed callbacks

Originally it was
```kotlin 
var color: Color by colorProperty
```
because it looks better when we use ```settings.color``` than when we write ```settings.colorProperty.value```, but due to future changes, this syntax variant caused an error (maybe something on the JVM level execution) and that has been replaced with this redundant realization

Reply:
In other classes this variant with ```by``` works at the same time. At the [torandofx guide](https://edvin.gitbooks.io/tornadofx-guide/content/part2/Property_Delegates.html) i've found that 

![image](https://user-images.githubusercontent.com/66365072/223710514-59e4aab7-b6db-4550-8a75-c0abe35fbd40.png)